test/rest.test.js

```diff
@@ -1184,6 +1184,27 @@ describe('strong-remoting-rest', function() {
           .expect(500)
           .end(expectErrorResponseContaining({message: 'an error'}, done));
       });
+      
+      it('should return 500 for unhandled errors thrown from before hooks', function(done) {
+        var method = givenSharedStaticMethod(
+          function greet(cb) {
+            cb();
+          }
+        );
+      
+        objects.before(method.name, function(ctx, next) {
+          process.nextTick(next);
+        });
+       
+        objects.before(method.name, function(ctx, next) {
+          throw new Error('test error');
+        });       
+       
+        request(app).get(method.url)
+          .set('Accept', 'application/json')
+          .expect(500)
+          .end(expectErrorResponseContaining({message: 'test error'}, [''], done));      
```

Comment:
Coding style: we put spaces inside curly brackets (objects). (But we don't put spaces inside square brackets for array). 

(I do realise the check on the line 1186 is not conforming to this rule, our coding style is unfortunately not very consistent ATM.)

Also the second argument `['']` is IMO not needed.

```
.end(expectErrorResponseContaining({ message: 'test error' }, done));
```